doc/sources/preview.rst

```diff
@@ -49,11 +49,11 @@ For example, you can set the environment variable in the following way:
 Then, you can import Scikit-learn estimator patched with a preview one from `sklearnex.preview` module::
 
      from sklearnex import patch_sklearn
-     patch_sklearn()
-     from sklearn.linear_model import Ridge
-     print(Ridge.__module__)
+     patch_sklearn(preview=True)
```

Comment:
```suggestion
     patch_sklearn()
```
arg should not needed due to env variable setting above

Reply:
yeah, did not notice env var setting in the file. quick question, why show customer a way to enable preview by setting env variable instead of explicit indication in code?

Reply:
Good point and I tend to agree on setting using code instead of env variable. Any objections to this @Alexsandruss ?

Reply:
I think it's better to replace env. variable setting with patching call but add note below for variable.